### DISCUSSION TITLE
Allow relative date filter

### DISCUSSION THREAD

**Discussion by franckleveque** [mealie-recipes/mealie#4717] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

In the recipe planner rules, i can't create a rule that exclude recipe already made in the last two weeks. I must set a specific date.

### Please provide a concise description of the feature that would resolve your issue.

It would great to have the choice to either configure a fixed date or configure a relative number of days from the date of the day.

### Please consider and list out some caveats or tradeoffs made in your design decision

The only caveat I see is the way to design the new component.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by elektroman-23** [comment]
I second this!
Would be great to create a rule that last made needs to be at least X days ago.

**Comment by michael-genson** [comment]
FYI, this will be implemented in https://github.com/mealie-recipes/mealie/pull/7034
